capture_service/dive_client_cli.cc

```diff
@@ -113,6 +121,27 @@ download_path,
 ".",
 "specify the full path to download the capture on the host, default to current directory.");
 
+ABSL_FLAG(
+std::string,
+device_architecture,
+"x86",
+"specify the device architecture to capture with gfxr. If not specified, the default is x86.");
```

Comment:
could you add the other valid options in here? (so that the user don't have the check code to find all the valid options :) )